pkg/services/queryhistory/queryhistory_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"io"
 	"net/http"
+	"net/url"
```

Comment:
The following is needed at the top of the file to run the integration tests with postgres and mysql: 

```go
//go:build integration
// +build integration

package queryhistory
```

Reply:
Updated in https://github.com/grafana/grafana/pull/45932/commits/7038a112e11dc11fbd24237bf34332bf8dda42fe